features/support/env.rb

```diff
@@ -20,3 +20,10 @@
     set_environment_variable('RBXOPT', "-Xint=true #{ENV['RBXOPT']}")
   end
 end
+
+Before('@kw-arguments') do
+  unless RSpec::Support::RubyFeatures.kw_args_supported?
```

Comment:
Wondering if `require 'rspec/support/ruby_features'` is needed here like we do e.g. [here](https://github.com/rspec/rspec-core/blob/640a9f62d954f69993b9e95511303dbda9b085e2/features/support/jruby.rb#L2).

Reply:
Its the skip logic thats broken for some reason

Reply:
I think its probably the ancient version of cucumber that the legacy specs run :joy: